gpu/Dockerfile

```diff
@@ -261,15 +261,14 @@ RUN echo "ESPnet build with CUDA" && \
     . ./setup_cuda_env.sh /usr/local/cuda;  \
     if [ "${CUDA_VER}" = "10.1" ]; then \
         # warpctc is not supported from Pytorch 1.3.1
-        MY_OPTS="${MY_OPTS} TH_VERSION=1.6.0";  \
+        MY_OPTS="${MY_OPTS} TH_VERSION=1.3.1";  \
     fi; \
     echo "Make with options ${MY_OPTS}" && \
     ln -s /kaldi ./ && \
     ./setup_anaconda.sh /miniconda espnet 3.7.4 && \
-    make KALDI=/kaldi ${MY_OPTS} -j 4
+    make KALDI=/kaldi ${MY_OPTS} -j`nproc`
 
-# There will be a warning saying that cuda *seems* not available in torch, but it will be: just not forget to run the image with "--gpus all" and it will be available.
-# NOTE: we can later remove the useless dependencies by applying a rm command on specific files inside tools folder.
+# There will be a warning saying that cuda *seems* not available in torch, but it will be: just not forget to run the image with "--gpus all" and it will be available. During docker building, nvidia-smi and cuda stuff are not fully available, but they are during runtime; the only way I found to force this visibility during building is described (here)[https://stackoverflow.com/questions/59691207/docker-build-with-nvidia-runtime], but I did not try it because it is quite invasive…
```

Comment:
> *seems* not available in torch, but it will be: just not forget to run the image

should be

> seems not to be available in torch, but it will be: just don't forget to run the image